apps/web/src/utils/applicationUtils.ts

```diff
@@ -157,11 +159,16 @@ export function applicationStepsToStepperArgs(
 export type Application = Omit<PoolCandidate, "user">;
 
 export function isApplicationInProgress(a: Application): boolean {
+  const isExpired = a.pool.closingDate
+    ? isPast(parseDateTimeUtc(a.pool.closingDate))
+    : false;
   return (
-    a.status === PoolCandidateStatus.Draft ||
-    a.status === PoolCandidateStatus.NewApplication ||
-    a.status === PoolCandidateStatus.ApplicationReview ||
-    a.status === PoolCandidateStatus.UnderAssessment
+    !isExpired &&
+    (a.status === PoolCandidateStatus.Draft ||
```

Comment:
I think it should only check for pool closing date if draft
Otherwise applications under review for pools get marked as past applications if the pool closed. But the closing date I think only matters for applying, so for draft

Reply:
changed in [1da3881](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6988/commits/1da3881c0adb11bd84b06e51b7627ecb69922153)